src/proteus/session/Session.js

```diff
@@ -316,11 +316,9 @@ class Session {
    * @returns {string}
    */
   _decrypt_cipher_message(envelope, msg) {
-    let state = this.session_states[msg.session_tag];
+    const state = this.session_states[msg.session_tag];
```

Comment:
Should stay `let` since in line [329](https://github.com/wireapp/proteus.js/blob/master/src/proteus/session/Session.js#L329) we are re-assigning `state`.

Reply:
Thanks